src/yang/drned-xmnr.yang

```diff
@@ -140,6 +140,12 @@ module drned-xmnr {
           "How long should XMNR wait for a test to do clean-up in case
            of an action abort.";
       }
+      leaf driver-file {
+        type string;
```

Comment:
The leaf is not used in tests, can you update them - at least `clined.lux` that tests also the import-convert action and that fails now?